harness/tests/experiment/fixtures/tf_keras_one_var_model.py

```diff
@@ -22,6 +22,7 @@ def make_one_var_tf_dataset_loader(hparams: Dict[str, Any], batch_size: int):
 
 
 class OneVarTrial(keras.TFKerasTrial):
+    _searcher_metric = "loss"
```

Comment:
I didn't see those `_searcher_metric` fields being used.

Reply:
Yeah, this was a bit hacky, it's used to automatically construct experiment configs with the proper searcher metric set.  That used to not be important because the workload manager did the searcher metric checks, and that used to not be in the unit tests.

```python
def make_trial_controller_from_trial_implementation(
    trial_class: Type[det.Trial],
    hparams: Dict,
    workloads: workload.Stream,
    scheduling_unit: int = 1,
    trial_seed: int = 0,
    exp_config: Optional[Dict] = None,
    checkpoint_dir: Optional[str] = None,
    latest_checkpoint: Optional[Dict[str, Any]] = None,
) -> det.TrialController:
    if not exp_config:
        assert hasattr(
            trial_class, "_searcher_metric"
        ), "Trial classes for unit tests should be annotated with a _searcher_metric attribute"
        searcher_metric = trial_class._searcher_metric  # type: ignore
        exp_config = make_default_exp_config(
            hparams, scheduling_unit, searcher_metric, checkpoint_dir=checkpoint_dir
        )
```

I thought it was better to annotate the test fixture models with an extra attribute rather than hand-code the searcher metric in the experiment configs everywhere that each model was referenced in tests.